debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/AbstractRecordsProducerTest.java

```diff
@@ -287,8 +290,19 @@
         expected.put("key1", "val1");
         expected.put("key2", "val2");
         expected.put("key3", "val3");
-        return Arrays.asList(new SchemaAndValueField("hs", hstoreMapSchema(), expected));
-        }
+
+        Map<String, String> expectedArray1 = new HashMap<>();
+        expectedArray1.put("key4", "val4");
+        expectedArray1.put("key5", null);
+
+        Map<String, String> expectedArray2 = new HashMap<>();
+        expectedArray2.put("key6", "val6");
+
+        return Arrays.asList(
```

Comment:
Do you test also `hstore.handling.mode` set to `json`?

Reply:
Added it. Note there's one potential issue with the stability of these tests: they rely on the order of entries in the map produced by `HStoreConverter`. More generally speaking, the order of HSTORE entries isn't guaranteed, so that JSON-based assertion might be prone to failures. That being said, it seems stable so far (hashCode of Strings is pretty stable after all), so I'd prefer to look into this if it actually proves to be a problem.